src/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import { AppBar, Toolbar, Stack, Typography, Button } from "@mui/material"
+import React from "react"
+
+export function Navbar() {
+  const navItems = ["Adopt", "Volunteer", "Donate", "About"]
```

Comment:
I like the array of strings for for the navbar entries, looks clean and functional. Great job!